### PR TITLE
Updated versions and CHANGELOG for the October 2020 Key Vault release.

### PR DESCRIPTION
**Pull request by vcolin7**

### CODE REVIEW

**Comment by vcolin7** [comment]
There's [a compilaton error](https://dev.azure.com/azure-sdk/public/_build/results?buildId=567525&view=logs&j=3d7a0212-45d0-5080-1981-6d6ef159b344&t=35a16916-ba13-5dc7-5456-9e14cf3ba1ca&l=23) in an unrelated module (azure-resourcemanager-keyvault). Not sure how this made it into master without failing in the pipeline.

**Comment by vcolin7** [comment]
The compilation error was due to a bad merge from master. Fixed it.

File: sdk/keyvault/azure-security-keyvault-certificates/CHANGELOG.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # Release History
 
-## 4.2.0-beta.1 (Unreleased)
-
+## 4.1.2 (2020-10-08)
+- Updated versions for azure-core and azure-identity.
```
Comment (g2vinay) [inline sdk/keyvault/azure-security-keyvault-certificates/CHANGELOG.md:4]: Follow new changelog guidelines:
CC: @samvaity

File: sdk/keyvault/azure-security-keyvault-keys/CHANGELOG.md [hunk 2]
```diff
@@ -1,7 +1,7 @@
 # Release History
 
-## 4.3.0-beta.1 (Unreleased)
-
+## 4.2.2 (2020-10-08)
+- Fixed an issue that prevented the `tags` and `managed` members of `KeyProperties` from getting populated when retrieving a single key using `KeyClient`, `KeyAsyncClient`, `CryptographyClient` and `CryptographyAsyncClient`.
```
Comment (g2vinay) [inline sdk/keyvault/azure-security-keyvault-keys/CHANGELOG.md:4]: Follow new Changelog guidelines.
CC:  @samvaity

File: sdk/keyvault/azure-security-keyvault-secrets/CHANGELOG.md [hunk 3]
```diff
@@ -1,7 +1,7 @@
 # Release History
 
-## 4.3.0-beta.1 (Unreleased)
-
+## 4.2.2 (2020-10-08)
+- Updated versions for azure-core and azure-identity.
```
Comment (g2vinay) [inline sdk/keyvault/azure-security-keyvault-secrets/CHANGELOG.md:4]: follow the new style of changelog updates.
Add a section called Dependency updates and mention this under that.

**Review by g2vinay — APPROVED** [review]

**Review by samvaity — APPROVED** [review]

**Pull request merged**